lib/sepia/src/util.js

```diff
@@ -129,7 +129,10 @@ function substituteWithOpaqueKeys(text) {
   var substitutions = globalOptions.substitutions;
   for (var i=0; i<substitutions.length; i++) {
     var subst = substitutions[i];
-    text = text.replace(subst.actualValueFn(), subst.opaqueKey);
+    var actualValue = subst.actualValueFn();
```

Comment:
I was worried that might affect backwards compatibility. It might not work if there were any special regex characters are being used.

```javascript
text = text.replace(new RegExp(subst.actualValueFn().replace(/[.*+?^${}()|[\]\\]/g, "\\$&"), 'g'), subst.opaqueKey);
```

😷